pkg/api/api.proto

```diff
@@ -64,6 +66,31 @@ message DeleteEnvironmentApplicationLockRequest {
   string lock_id = 3;
 }
 
+
+message CreateNewReleaseRequest {
+  string environment = 1;
+  string application = 2;
+  optional string team = 3;
+  // env->yaml
+  map<string, string> manifests = 4;
+  optional uint64 version = 5;
```

Comment:
I would avoid the three optionals here. We are already using 0 for "no version supplied" further downstream and empty strings aren't valid values for team and soure repo anyway.

Reply:
I'm fine with that as long as we don't expose the grpc endpoint.

If we do expose it, I would like to have clear indications of what is optional and what is not.

Reply:
Solved it by not exposing it as grpc in the frontend service